src/utils/project-content/get-project-exports.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path';
+
+import { IProject } from '../../../types';
+
+interface Export {
+  slug: string;
+  path: string;
+}
+
+export const getProjectExports = (projectContent: IProject): Export[] =>
+  projectContent.collections
+    .map((collection) =>
```

Comment:
Only since 11 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap

We can change it on April 2021 :sweat_smile: